tests/kubernetes-agent/KubernetesAgent.IntegrationTests/Setup/ServerHalibutRuntimeRunner.cs

```diff
@@ -0,0 +1,27 @@
+using Halibut;
+using Halibut.Diagnostics;
+using KubernetesAgent.Integration.Setup.Common;
+
+namespace KubernetesAgent.Integration.Setup
+{
+    public class ServerHalibutRuntime
```

Comment:
Is this used?

Reply:
No - it was a part of a refactor I was going to do to make it nicer to read, but didn't end up finishing! Removed!